test/js/test.js

```diff
@@ -456,14 +384,13 @@ function getReadyToValidate(err, resp, body) {
       response.end("");
     }
   }
+  state.httpServer = https.createServer({
+    cert: fs.readFileSync("temp-cert.pem"),
+    key: fs.readFileSync(state.keyFile)
```

Comment:
FWIW, in Node/browsers newer than IE 8, you can use trailing commas in dictionaries. would make the style consistent with the Go code, anyway.